test/qtismtest/data/UtilsTest.php

```diff
@@ -147,4 +148,48 @@ public function testgetLastItem2()
         $this->assertEquals(null,
             DataUtils::getLastItem($test, $test->getComponentByIdentifier('S96'), $sections));
     }
+
```

Comment:
All the tests related to the Format class should be written in qtismtest\common\utils\FormatTest class!